platform/broadcom/sonic-platform-modules-ruijie/common/modules/i2c-mux-pca954x.c

```diff
@@ -0,0 +1,1763 @@
+/*
+ * Copyright (c) 2008-2009 Rodolfo Giometti <giometti@linux.it>
```

Comment:
can you use the one in linux kernel?

Reply:
Will revise I2C deadlock problem when use 9548 and 9541 at the same time.